src/app/results/results.component.css

```diff
@@ -709,7 +720,7 @@ div.autocorrect {
   }
 
   #search-options li {
-    padding: 0 4vw 12px;
+    padding: 0 3vw 12px;
```

Comment:
Either use ```vw``` or ```px```

Reply:
px is used for fixed space at bottom and vw is used for relative spacing at right between the tab.

Reply:
px can do it same?

Reply:
We want fixed padding at bottom and relative padding at right.

Reply:
Could you fix Codacy?